genai-perf/genai_perf/config/generate/perf_analyzer_config.py

```diff
@@ -63,14 +273,33 @@ def get_parameters(self) -> Dict[str, Any]:
         """
         return self._parameters
 
+    def get_obj_args(self) -> Namespace:
```

Comment:
Is the new method used in the code?

Reply:
Not in this code, but it is used in the main code that runs analyze live.